core/src/main/java/org/kohsuke/stapler/Function.java

```diff
@@ -94,9 +94,11 @@
     boolean bindAndInvokeAndServeResponse(Object node, RequestImpl req, ResponseImpl rsp, Object... headArgs) throws IllegalAccessException, InvocationTargetException, ServletException, IOException {
         try {
             Object r = bindAndInvoke(node, req, rsp, headArgs);
-            if (getReturnType()!=void.class)
-                renderResponse(req,rsp,node, r);
+            if (getReturnType() != void.class)
+                renderResponse(req, rsp, node, r);
```

Comment:
come on @jglick !!! you know perfectly that these little changes improve the readability of the source code.

I know also, you can review this PR even if it has obfuscated :dizzy_face: